src/Pages/login/login.jsx

```diff
@@ -64,30 +66,37 @@ export default () => {
           </div>
 
           <div className="w-full">
-            <input
+            <InputField
               type="password"
               placeholder="Password"
+              value={password}
               onBlur={() => {
                 setPasswordActivated(true);
-
-                checkPassword(password);
+                checkPassword();
               }}
-              className="border p-1 rounded-md border-Gray drop-shadow w-full"
-              value={password}
-              onChange={(e) => setPassword(e.target.value)}
+              onchange={(e) => setPassword(e.target.value)}
             />
-
             {isPasswordActivated && (
               <div>
                 <p className="text-red text-xs">{passwordAlert}</p>
               </div>
             )}
           </div>
         </div>
+
+        {!isInputsValid && isLoginClicked && (
+          <div>
+            <p className="text-red text-xs">Invalid Username or Password</p>
+          </div>
+        )}
+
         <div>
           <button
             className="px-7 py-4 text-white  bg-button rounded-full hover:bg-buttonHover active:bg-buttonHover"
-            onClick={handleClick}
+            onClick={() => {
+              setLoginClicked(true);
```

Comment:
This is also another instance of separating code. We can directly move this line into the `handleLogin` function.

Reply:
Fixed in 925d48b0a98fb7bd42aa97c5a80d13fa92be9888